src/components/modules/RecentAndTrending/index.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { PureComponent } from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { ModuleBox, OverlapScaffold, Post, Pattern } from '@/components/modules';
+import { Container, Row, Image } from '@/components/primitives';
+import { Responsive } from '@/components/utils';
+import { BREAKPOINTS, CONTAINER_PADDING, COLOR_MAP } from '@/constants';
+import { TagTileBoxModuleData, PostType, OFFSET_DIRECTION, GridOrder } from '@/types';
+import PostGrid  from './PostGrid';
+import TrendingLine from '@/assets/images/trending-line.svg';
+
+interface Props extends TagTileBoxModuleData {
+    postsOne: PostType[];
+    postsTwo: PostType[];
+    sectionLink: string;
+    sectionColor: string;
+    sectionTitle: string;
+    order?: GridOrder;
+    tag: string;
+}
+
+export default class RecentAndTrending extends PureComponent<Props> {
+    scaffoldProps = (isDesktop: boolean) =>
+    isDesktop
+      ? {
+          containerPadding: 45,
+          overlap: 30
+        }
+      : {
+          containerPadding: CONTAINER_PADDING.MOBILE,
+          overlap: 15
+        };
+
+    MainPostImageProps = (isDesktop: boolean) =>
+    isDesktop
+      ? {
+          imageWidth: 1200,
+          imageHeight: 627
+        }
+      : {
+          imageWidth: 375,
+          imageHeight: 250
+        };
+
+
+    render() {
+        const {
+            sectionColor,
+            postsOne: [mainPost, ...secondaryPosts],
+            postsTwo, 
+            tag
+        } = this.props;
+
+        return (
+            <Responsive>
+            {({ minWidth }) => {
+                const isDesktop = minWidth(BREAKPOINTS.LG);
+
+                const localFontStyles = isDesktop
+                    ? { fontSize: 24, lineHeight: 29 }
+                    : { fontSize: 20, lineHeight: 24 };
+                return (
+                    <Container type='content' style={{flexDirection: isDesktop? 'row' : 'column', justifyContent: 'space-between'}}>
+                      <View style={{width: isDesktop ? '68%' : '100%'}}>
+                        <Text style={[styles.headerText, localFontStyles, {marginLeft: isDesktop ? 0 : '15px'}]}>Recently in {capitalize(tag)}</Text>
```

Comment:
This view could be encapsulated as a Recent component

Reply:
Moved to its own React component